spec/models/users_spec.rb

```diff
@@ -8,6 +8,11 @@
   it { should have_many(:queue_items).order("position") } 
   it { should have_many(:reviews).order("created_at DESC") }
 
+  it "returns nil for user token when not issuing password reset requests" do
+    alice = Fabricate(:user)
+    expect(alice.token).to be_nil
```

Comment:
I can not see any trigger in this spec. It should always be:

1. setup,
2. trigger,
3. expectation/assertion.